crates/win-api-wrappers/src/wts.rs

```diff
@@ -20,78 +19,77 @@ use crate::ui::MessageBoxResult;
 use crate::utils::{AnsiString, SafeWindowsString};
 
 /// RAII wrapper for WTS virtual channel handle.
-pub struct WTSVirtualChannel(HANDLE);
+pub struct WtsVirtualChannel(HANDLE);
 
-impl WTSVirtualChannel {
+impl WtsVirtualChannel {
     /// # Safety
-    /// `handle` must be a valid handle returned from `WTSVirtualChannelOpenEx`.
-    pub unsafe fn new(handle: HANDLE) -> Self {
+    ///
+    /// `handle` must be a valid handle returned by `WTSVirtualChannelOpenEx`.
+    unsafe fn new(handle: HANDLE) -> Self {
         Self(handle)
     }
 
     pub fn open_dvc(name: &str) -> anyhow::Result<Self> {
         let channel_name = AnsiString::from(name);
 
-        // SAFETY: Channel name is always a valid pointer to a null-terminated string.
-        let raw_wts_handle = unsafe {
-            WTSVirtualChannelOpenEx(WTS_CURRENT_SESSION, channel_name.as_pcstr(), WTS_CHANNEL_OPTION_DYNAMIC)
-        }?;
+        // SAFETY: `channel_name.as_pcstr()` is always a valid pointer to a null-terminated string.
+        let wts_handle = unsafe {
+            WTSVirtualChannelOpenEx(WTS_CURRENT_SESSION, channel_name.as_pcstr(), WTS_CHANNEL_OPTION_DYNAMIC)?
```

Comment:
Question(style): I was always wondering, which is the best -- to place `?` operator inside `unsafe` block or outside. I usually lean towards placing it outside, because `try` operation by itself is not unsafe, and thus could be moved out -- however from a readability standpoint it looks better if `?` is inside the block. What is your spin on this, @CBenoit ?

Reply:
I think it’s better to keep the ? inside the unsafe block, just at the end of the function call. That’s how we usually do in safe Rust, and as you said, the `try` operation is not unsafe, and will never be, so we’re probably not going to be confused about. We also have the `clippy::multiple_unsafe_ops_per_block` lint to avoid inadvertently calling more than one unsafe operation in the same block. If we follow the logic to put the `?` outside, then why not put things like `as_pcstr()` outside as well? I think it would be too much :thinking:
Of course there is a balance to strike for. In this case, the outermost function `WTSVirtualChannelOpenEx` is (I think) obviously the unsafe operation.
But when we perform pointer dereferencing, it may be worth spilling on multiple lines.

Reply:
> If we follow the logic to put the ? outside, then why not put things like as_pcstr() outside as well? I think it would be too much

Fair enough 👍 

> But when we perform pointer dereferencing, it may be worth spilling on multiple lines.

Yep, agree. I think Clippy will catch that as multiple unsafe operations in a single `unsafe` block iirc